build_tools/docker/README.md

```diff
@@ -38,42 +38,28 @@ dependencies based on these image names.
 We use a helper python script to manage the Docker image deployment. It lists
 all images and their dependencies and manages their canonical registry location.
 When creating a new image, add it to this mapping. To build an image and all
-images it depends on:
+images it depends on as well as pushing them to GCR and updating all references
+to the image digest. Pushing images to GCR requires the `Storage Admin` role in
+the `iree-oss` GCP project.
```

Comment:
Add a note about needing to request this for OSS contributors?

We might also want to include a brief section on pulling and running images, since that isn't possible via the script without these permissions. Something like:


## Running Images Locally

Production versions of the images can be download and run locally as follows:

```shell
PROD_IMAGE_URL="gcr.io/iree-oss/bazel-tensorflow-swiftshader:prod"
docker pull "${PROD_IMAGE_URL?}"
docker run -it --rm --entrypoint bash "${PROD_IMAGE_URL?}"
```

Reply:
Done